src/main/java/de/dennisguse/opentracks/ShowErrorActivity.java

```diff
@@ -0,0 +1,86 @@
+package de.dennisguse.opentracks;
+
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.Menu;
+import android.view.MenuItem;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import java.io.UnsupportedEncodingException;
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+
+import de.dennisguse.opentracks.databinding.ActivityShowErrorBinding;
+
+public class ShowErrorActivity extends AppCompatActivity {
+
+    public static final String EXTRA_ERROR_TEXT = "error";
+
+    private static final String TAG = ShowErrorActivity.class.getSimpleName();
+
+    private ActivityShowErrorBinding binding;
+
+    @Override
+    public void onCreate(final Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        binding = ActivityShowErrorBinding.inflate(getLayoutInflater());
```

Comment:
Why not inflate in `getRootView()`?
PS/ don't what the difference actually is...

Reply:
Ah, right. OT has it's own AbstractActivity with `getRootView()`, I'll change it.